client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -133,23 +143,26 @@ const ProductLightbox: React.FC< Props > = ( {
 				</div>
 				<div className="product-lightbox__variants">
 					<div className="product-lightbox__variants-content">
-						<div className="product-lightbox__variants-options">
-							<MultipleChoiceQuestion
-								question="Choose a storage option:"
-								answers={ [
-									{ id: '10GB', answerText: '10GB' },
-									{ id: '1TB(1,000GB)', answerText: '1TB(1,000GB)' },
-								] }
-								selectedAnswerId={ checked }
-								onAnswerChange={ setChecked }
-								shouldShuffleAnswers={ false }
-							/>
-						</div>
-						<PaymentPlan isMultiSiteIncompatible={ isMultiSiteIncompatible } />
+						{ shouldShowOptions && (
+							<div className="product-lightbox__variants-options">
+								<MultipleChoiceQuestion
+									question={ `${ translate( 'Choose a storage option' ) }:` }
```

Comment:
`question` needs to be based on the product and its variants. For example, for Jetpack Social, it needs to be `Choose a number of shares/month:`

Reply:
Hmm, thanks for spotting this. Could you please navigate to these requirements? Maybe I missed these.

Reply:
You can check Figma for lightbox data
m0y1jHTuDpvMPkFsOJBoNP-fi-3557%3A123824

Reply:
What @manzoorwanijk is referring to can be seen here: p1HpG7-hbV-p2

Here's the mockup image:

![Markup 2022-09-13 at 14 10 36](https://user-images.githubusercontent.com/11078128/189978549-a9886d85-fde8-4aae-8c72-b1896d8b6862.png)

However, I don't think Jetpack Social has additional product variants (with variant shares per month) added to the WPCOM store quite **yet**. Right now there is only Jetpack Social Basic (monthly & yearly). But still, at some point in the near future, `question` will be variable, depending on the product.